data/json/items/gun/223.json

```
@@ -764,5 +764,79 @@
     "pocket_data": [
       { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "augmag_30rd", "augmag_10rd", "augmag_42rd", "augmag_100rd" ] }
     ]
+  },
+  {
+    "id": "zpapm90",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "ZPAP M90" },
+    "description": "Civilian AK pattern rifle made by Zastava in 5.56 NATO.",
+    "weight": "3402 g",
+    "volume": "2167 ml",
+    "longest_side": "99 cm",
+    "price": 79500,
+    "price_postapoc": 3000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "ammo": [ "223" ],
+    "dispersion": 180,
+    "durability": 8,
+    "min_cycle_recoil": 1350,
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
+      [ "stock accessory", 2 ],
+      [ "underbarrel mount", 1 ]
```

```suggestion
      [ "rail", 2 ],
      [ "sights mount", 1 ],
      [ "sling", 1 ],
      [ "stock", 1 ],
      [ "stock accessory", 2 ],
      [ "underbarrel", 1 ]
```
https://zastavaarmsusa.com/product/polymer-furniture-zpapm90ps/

> The modular design includes a smooth top cover as well as **three short sections of Picatinny rail** that can be attached to the sides and bottom of the lower portion for accessories.